### DISCUSSION TITLE
Is it possible to create multiple pages of a PDF at the same time?

### DISCUSSION THREAD

**Discussion by saito-eiji** [pdfme/pdfme#72] [Q&A] [answered]
Hello.

Is it possible to create multiple pages of PDF at the same time when there is a lot of data to put on the page?

**Comment by hand-dot** [comment, marked as answer]
Hi @saito-eiji 
Yes, You can.  Please select a multiple-page PDF from "Change BasePDF" of https://pdfme.com/template-design .
then you will make sense.

**Comment by saito-eiji** [comment]
Hi @hand-dot 

Thank you for your reply.
One question.
Is it possible to increase the number of pages dynamically (based on the number of data entered) in the front?

**Reply by hand-dot** [reply]
@saito-eiji 
yes, but a little bit tricky...
you have to change basePdf dynamically.

so, you have to prepare some pattern of basePdf for depending on the input quantity

**Reply by saito-eiji** [reply]
@hand-dot 

I see.
Thank you for your instruction.
I will try to implement it!

**Reply by glaysonsantos-fiepb** [reply]
@hand-dot can you give us an example of doing that? 

I am struggling to create a basic PDF that grows dynamically based on the amount of char/data that I set as input. I want to get some API data and generate the PDF with them, but I don't know the amount of data the API will return me. Sometimes it can be just a few, but sometimes can be a lot. All I want to do is to page break when the content reaches the page limits, so the overflowing content sits perfectly in the next page.

Thanks a lot for this lib, it's amazing!
